### PR TITLE
[disasters] add additional date filters

### PR DESCRIPTION
**Pull request by chejennifer**
- add additional "last 30 days" and "last 6 months" date options
- change fetching data for a specific date to fetching data for a date range

Screenshot 2022-12-29 at 2 39 41 PM

### CODE REVIEW

File: static/js/components/tiles/disaster_event_map_tile.tsx [hunk 1]
```diff
@@ -28,14 +28,18 @@ import {
...
+} from "../../constants/disaster_event_map_constants";
 import {
   EUROPE_NAMED_TYPED_PLACE,
   IPCC_PLACE_50_TYPE_DCID,
   USA_PLACE_DCID,
 } from "../../shared/constants";
 import { NamedPlace, NamedTypedPlace } from "../../shared/types";
 import { loadSpinner, removeSpinner } from "../../shared/util";
-import { getAllChildPlaceTypes } from "../../tools/map/util";
+import { getAllChildPlaceTypes, getDate } from "../../tools/map/util";
```
Comment (juliawu) [inline static/js/components/tiles/disaster_event_map_tile.tsx:42]: what does getDate get used for?

File: static/js/components/tiles/disaster_event_map_tile.tsx [hunk 2]
```diff
@@ -90,8 +95,9 @@ export function DisasterEventMapTile(
   const svgContainerRef = useRef(null);
   const infoCardRef = useRef(null);
   const europeanPlaces = useRef([]);
+  const dateRanges = useRef(getDateRanges());
   const [dateList, setDateList] = useState([]);
-  const [selectedDate, setSelectedDate] = useState("");
+  const [selectedDate, setSelectedDate] = useState(DATE_OPTION_30D_KEY);
```
Comment (juliawu) [inline static/js/components/tiles/disaster_event_map_tile.tsx:100]: Maybe we should use DATE_OPTION_6M_KEY as default for now? There's no data for the last 30 days currently.

**Review by juliawu — COMMENTED** [review]
LGTM! Just some quick comments:

**Review by juliawu — APPROVED** [review]

File: static/js/components/tiles/disaster_event_map_tile.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/components/tiles/disaster_event_map_tile.tsx]: good catch, it is unused, removed now

File: static/js/components/tiles/disaster_event_map_tile.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/components/tiles/disaster_event_map_tile.tsx]: good idea, updated

**Pull request merged**